lib/streamlit/elements/dialog_decorator.py

```diff
@@ -133,8 +142,8 @@ def dialog_decorator(
         fragments, and fragments can't contain dialogs. Using dialogs in widget
         callback functions is not supported.
```

Comment:
nit: need to update this docstring to remove the limitations about dialogs not being able to call fragments + not being trigger-able by widget callbacks

Reply:
oh yeah great catch, thanks!